### ISSUE TITLE
ToDos in ApRES

### ISSUE THREAD

**Issue by benhills** [dlilien/ImpDAR#26]
These are scattered throughout the ApRES scripts. 
I need to address them all and am opening this issue so that I am sure to come back to it.

**Comment by dlilien** [comment]
Sorry for the delay--I finally got around to merging this. There are some lingering issues with testing that are due to changes by github--I will get those fixed, then put this into main.

**Comment by benhills** [comment]
Yep, no worries. I am thinking about building out 'apdar' quite a lot more before the field season this year so that the folks joining us have an easier time processing the pres data quickly (I am going to be mostly hands off on pres this year I think). 

Then, I am undecided on whether polarimetry should be included in apdar or should be a separate command, 'qpdar' or something.

My point is, more PRs will be coming and we could wait to put them into main all at the same time, I am not in a rush.

**Comment by dlilien** [comment]
Ok, we can wait on polarimetry, but all this is now merged into main and published to PyPi (and devel is up to date as well).
